### PR TITLE
Migrate snekclientv2 aa1 to implement 1

### PR DESCRIPTION
**Pull request by schettn**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently [SnekClient V2](https://github.com/snek-at/client) is not supported. 😞 

**What is the new behavior (if this is a feature change)?**
- Now  [SnekClient V2](https://github.com/snek-at/client) is supported. 😃 

**Other information**:
- 🐍

### CODE REVIEW

File: src/store/actions/authActions.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const loginAction = (user) => {
       return session
         .begin(user)
         .then((whoami) => {
-          if (whoami?.username !== process.env.REACT_APP_ANONYMOUS_USER) {
+          if (!whoami?.anonymous) {
```
Comment (schettn) [inline src/store/actions/authActions.js]: Note to myself..:

What if whoami is undefined?

Comment (schettn) [reply src/store/actions/authActions.js]: image

This is issue is not urgent cause whoami cannot be undefined.
However we should adjust it anyways.

Comment (kleberbaum) [inline src/store/actions/authActions.js]: This line is absolutely beautiful but probably a bug.

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

**Review by pinterid — APPROVED** [review]
LGTM

**Review by Aichnerc — APPROVED** [review]
LGTM

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**